src/windows/enumerate.rs

```diff
@@ -345,8 +346,7 @@ impl PortDevice {
         // Using the unicode version of 'SetupDiGetDeviceRegistryProperty' seems to report the
         // entire mfg registry string. This typically includes some driver information that we should discard.
         // Example string: 'FTDI5.inf,%ftdi%;FTDI'
-        String::from_utf16_lossy(&property_buf)
-            .trim_end_matches(0 as char)
+        from_utf16_lossy_trimmed(&property_buf)
```

Comment:
Don't we have to pay attention to buffer versus pointer type and the actually returned type indicated by `SetupDiGetDeviceRegistryPropertyW` as for `RegEnumValueW` as well?

Reply:
Sorry for the confusion! I did not manage to annotate farther away from your actual change. 😅 I mean the call to `SetupDiGetDeviceRegistryPropertyW` which also provides type information for the returned value and I'm proposing changing this with 727273e.

Reply:
I think I added a matching check? Not sure if comprehensive though